File: 2221-find-triangular-sum-of-an-array/2221-find-triangular-sum-of-an-array.cpp
class Solution {
public:
    int triangularSum(vector<int>& nums) {
        while (nums.size() > 1) {
            std::vector<int> res;
            for (int i = 0; i < nums.size() - 1; ++i) {
                res.push_back((nums[i] + nums[i + 1]) % 10);
            }
            nums = res;
        }
        return nums[0];
        
    }
};

File: 2221-find-triangular-sum-of-an-array/2221-find-triangular-sum-of-an-array.py
class Solution:
    def triangularSum(self, nums: List[int]) -> int:
        while len(nums)>1:
            res=[]
            for i in range(len(nums)-1):
                res.append((nums[i]+nums[i+1])%10)
            nums[:]=res[:]
        return nums[0]